dvc/command/experiments.py

```diff
@@ -471,9 +473,21 @@ def show_experiments(
         }
     )
 
-    if kwargs.get("only_changed", False):
+    if kwargs.get("only_changed", False) or html:
         td.drop_duplicates("cols")
 
+    extra_render_args = {}
+    if html:
+        td.dropna("rows")
+        td.column("Experiment")[:] = [
```

Comment:
@dberenbaum continuing discussion here because it is easier for me to track parallel conversations on threaded comments than on P.R. level comments.

> Does it make sense to always/ever include the experiment names? Is it possible to make a tooltip hover with experiment names? I think it can get distracting with lots of experiments:
> 
> ![image](https://user-images.githubusercontent.com/2308172/142533134-e6ed40a8-f401-4a43-9036-91857d6aa8ca.png)
> 
> To me, the main point of parallel coordinates plots is when there are too many to look at each individual experiment, so I wouldn't usually expect to see each experiment name.

Looks like hovering on lines of these plots is not supported nor in the plans to implement unless a sponsor funds the feature (https://github.com/plotly/plotly.js/issues/3012).

Assuming hover is not an option, would you think that adjusting the height of the plot so that the labels are not that messy would help? 

I will also try to find some other alternatives

Reply:
> Assuming hover is not an option, would you think that adjusting the height of the plot so that the labels are not that messy would help?

does it make to sponsor / work with them together?

Reply:
> Genuine question: do you think showing dozens of experiment names can be helpful? I wouldn't find it useful and would find differentiating the colors of each difficult, but maybe that's just me.

**Without** on hover tooltip, I would say that it is still better to have an awkward column (and work around making it less awkward) than not having a way to identify each individual line. **With** on hover tooltip, I would drop the `Experiment` column, no doubts.

> does it make to sponsor / work with them together?

It could be a worthy investment if we consider the underlying javascript component (where the changes should be made) to be reusable by other tools / products (i.e. `vscode`, `Studio`). 

Alternatively, if we consider this feature a blocker, we could explore other open-source alternatives for rendering the plot like: https://github.com/facebookresearch/hiplot . The renderer might be a little more tricky to implement but I think it would still be doable.

Reply:
Resolving after offline discussion. We would keep the column as it is for now and discuss options for improving the line-experiment association in https://github.com/iterative/dvc/issues/7080